bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/navbar/NavbarDropDownButton.java

```diff
@@ -66,23 +61,7 @@ protected final void addButtonBehavior(final ButtonBehavior buttonBehavior) {
     @Override
     protected WebMarkupContainer newButton(String markupId, IModel<String> labelModel, final IModel<IconType> iconTypeModel) {
         WebMarkupContainer button = super.newButton(markupId, labelModel, iconTypeModel);
-        button.add(new CssClassNameRemover("btn"));
-        button.add(new CssClassNameAppender(Buttons.Type.Menu.cssClassName()));
+        button.add(new CssClassNameAppender(Buttons.Type.NavLink.cssClassName()));
         return button;
     }
-
-    @Override
-    protected ButtonList newButtonList(String markupId) {
```

Comment:
I see design issue here: every class which will override `newButtonList` will have `dropdown-item` CSS class dropped :(

I had in plans to fix it as well ...

Reply:
Ah I see, that's why you moved this to `ButtonList::configureLink`. Nevertheless the design issue was there before and after realising this code is "useless" in the sense `Buttons.Type.Menu` is not providing anything meaningful anymore, I opted to drop it entirely.

Perhaps you can come up with a better design after the main issues are fixed?

Reply:
sure, it is not locker here
I wrote it to not forget :) (sort of "notes to myself" :)) )